cpu/nrf51822/include/nrfmin.h

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright (C) 2015 Freie Universität Berlin
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup        cpu_nrf51822_nrfmin NRF Minimal Radio Driver
+ * @ingroup         cpu_nrf51822

```

Comment:
I know, but for consistency I would put it in drivers because that is where I would look for such a driver in the documentation. Other driver implementations are part of CPU as well (e.g. every peripheral driver).